### PR TITLE
Add basic support for non-image attachments

### PR DESCRIPTION
**Pull request by paraketsova**
Non-image attachments show up as links under the images:

image

### CODE REVIEW

**Comment by paraketsova** [comment]
image

File: components/PostAttachments.js [hunk 1]
```diff
@@ -1,26 +1,46 @@
...
 
+      {otherAttachments.map(attId => (
+        <AttachmentOther id={attId} key={attId}/>
+      ))}
+
       <style jsx>{`
         section {
           grid-area: attachments;
-
+
```
Comment (clbn) [inline components/PostAttachments.js]: Let's remove these spaces (don't remove the newline tho).

File: components/PostAttachments.js [hunk 2]
```diff
@@ -1,26 +1,46 @@
...
 import AttachmentImage from './AttachmentImage';
+import AttachmentOther from './AttachmentOther';
 
 const PostAttachments = ({ postId }) => {
   const attachmentIds = useSelector(state => state.posts[postId].attachmentIds);
+
   if (attachmentIds.length === 0) {
     return false;
   }
+
```
Comment (clbn) [inline components/PostAttachments.js]: Please remove these new empty lines in `PostAttachments`.

File: components/AttachmentOther.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  return (
+    <a href={url} title={fileSize + ' b'} target="_blank" rel="noopener">
+      {fileName}
+
+      <style jsx>{`
+        a {
+          display: block;
+          margin-bottom: 0.5rem;
+        }
+
```
Comment (clbn) [inline components/AttachmentOther.js]: Extra line here.

**Review by clbn — CHANGES_REQUESTED** [review]
LGTM, except for some nitpicking on extra spaces.

File: components/PostAttachments.js [hunk 4]
```diff
@@ -1,26 +1,46 @@
...
   return <PostAttachmentsNotEmpty attachmentIds={attachmentIds}/>
 };
 
 const PostAttachmentsNotEmpty = ({ attachmentIds }) => {
+  const attachments = useSelector(state => state.attachments);
+  const imageAttachments = [];
+  const otherAttachments = [];
+
+  for (let attachmentId of attachmentIds) {
```
Comment (clbn) [inline components/PostAttachments.js]: Also, I think this `let` should be `const`.

File: components/PostAttachments.js [hunk 5]
```diff
@@ -1,26 +1,46 @@
...
+    if (attachments[attachmentId].mediaType && attachments[attachmentId].mediaType === 'image') {
```
Comment (clbn) [inline components/PostAttachments.js]: I don't think the first condition adds anything here, the second one is enough.

**Review by clbn — APPROVED** [review]
Thanks, looks good!

**Pull request merged**